resources/definitions/voron2_base.def.json

```diff
@@ -0,0 +1,155 @@
+            "ultimaker_tough_pla_black",
+            "ultimaker_tough_pla_green",
+            "ultimaker_tough_pla_red",
+            "ultimaker_tough_pla_white",
+            "ultimaker_tpu_black",
+            "ultimaker_tpu_blue",
+            "ultimaker_tpu_red",
+            "ultimaker_tpu_white"
+        ]
+    },
+    "overrides":
+    {
+        "machine_name":                                 { "default_value": "VORON2" },
+        "machine_width":                                { "default_value": 250 },
+        "machine_depth":                                { "default_value": 250 },
+        "machine_height":                               { "default_value": 250 },
+        "gantry_height":                                { "value": 30 },
+        "machine_heated_bed":                           { "default_value": true },
+        "machine_max_acceleration_x":                   { "default_value": 1500 },
+        "machine_max_acceleration_y":                   { "default_value": 1500 },
+        "machine_max_acceleration_z":                   { "default_value": 250 },
+        "machine_acceleration":                         { "default_value": 1500 },
+        "machine_max_jerk_xy":                          { "default_value": 20 },
+        "machine_max_jerk_z":                           { "default_value": 1 },
+        "machine_max_jerk_e":                           { "default_value": 60 },
+        "machine_steps_per_mm_x":                       { "default_value": 80 },
+        "machine_steps_per_mm_y":                       { "default_value": 80 },
+        "machine_steps_per_mm_z":                       { "default_value": 400 },
+        "machine_endstop_positive_direction_x":         { "default_value": true },
+        "machine_endstop_positive_direction_y":         { "default_value": true },
+        "machine_endstop_positive_direction_z":         { "default_value": false },
+        "machine_feeder_wheel_diameter":                { "default_value": 7.5 },
+        "machine_head_with_fans_polygon":               { "default_value": [ [-35, 65], [-35, -50], [35, -50], [35, 65] ] },
+        "machine_max_feedrate_z":                       { "default_value": 40 },
+        "machine_max_feedrate_e":                       { "default_value": 120 },
+        "machine_gcode_flavor":                         { "default_value": "RepRap (RepRap)" },
+        "machine_start_gcode":                          { "default_value": "print_start" },
+        "machine_end_gcode":                            { "default_value": "print_end" },
+        "adhesion_type":                                { "default_value": "skirt" },
+        "skirt_brim_minimal_length":                    { "default_value": 550 },
+        "retraction_speed":                             { "default_value": 80, "maximum_value_warning": 130 },
+        "retraction_retract_speed":                     { "default_value": "retraction_speed", "maximum_value_warning": 130 },
```

Comment:
Please take a look at the check result.
```
E       AssertionError: Unnecessary default_values for ['retraction_retract_speed'] in /__w/Cura/Cura/tests/Settings/../../resources/definitions/voron2_base.def.json
```

Reply:
Seems like this override was done to change the warning value. @Fulg, please confirm

Reply:
Yes, the intent was to raise the warning, since the default speed is already above the allowed maximum. I tested without it and the updated default value worked fine.